project.yaml

```diff
@@ -890,7 +890,7 @@ Layer:
             CASE WHEN int_surface IN ('unpaved') THEN 0 ELSE 2 END
         ) AS roads_casing
     properties:
-      minzoom: 10
+      minzoom: 9
```

Comment:
We should avoid having both high zoom and low zoom layers active at the same zoom level.

This may mean rendering from the high zoom layer in the same style as the low zoom layer.